src/org/pentaho/reporting/platform/plugin/cache/DeleteOldOnAccessCache.java

```diff
@@ -26,11 +26,7 @@
 import java.io.Serializable;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Star imports are frowned upon by our coding standard. Please configure your IDE to avoid those.